packages/networks/src/networks/vNaga/shared/managers/LitChainClient/apis/highLevelApis/mintPKP/MintPKPSchema.ts

```diff
@@ -17,8 +17,10 @@ export const MintPKPSchema = z
 
     // Determine pubkey based on the (potentially derived) authMethodType
     if (data.authMethodType === AUTH_METHOD_TYPE.WebAuthn) {
-      if (!data.pubkey) {
-        throw new Error('pubkey is required for WebAuthn');
+      if (!data.pubkey || data.pubkey === '0x') {
```

Comment:
`MintPKPSchema` here seems to be a fourth copy of `AuthServiceMintRequestSchema` -- shouldn't all of 4 these copies actually be a single schema that is exported from our either the schemas package or the lit client package so that callers can just use it directly and be sure that the schema is in sync with what the `mintPKP` method will actually apply when we call it?

Actually, it looks like the `MintPKPSchema` is incorrect, too -- it's showing `authMethodType` as being a union of bigint or number, but it should be constrained by the values of `AUTH_METHOD_TYPE` -- and we're doing a hard test here using `===` on `AUTH_METHOD_TYPE.WebAuthn`, but if someone passes in a bigint version it won't match, but will pass schema validation...